### PR TITLE
Add disabled to Pagination button

### PR DESCRIPTION
**Pull request by scouillard**
## Description

A bit of an annoying fix:

1. Semantic-ui pagination buttons are a button inside of an anchor. So we need to disable the anchor with `aria-disabled=true` in the Pagination component. 
image

2. We need to disable the button itself in PaginationButton component.
3. We need to overwrite the custom button in app.scss

## Screenshots (if appropriate):

image

### CODE REVIEW

File: app/javascript/components/shared_components/Pagination.jsx [hunk 1]
```diff
@@ -1,13 +1,27 @@
...
 export default function Pagination({
   page, totalPages, setPage,
 }) {
   const handlePage = (e, { activePage }) => setPage(activePage);
 
+  const disabledPrevious = () => {
+    if (page === 1) {
+      return true;
+    }
+    return false;
```
Comment (farhatahmad) [inline app/javascript/components/shared_components/Pagination.jsx:15]: you can just say `return page ===1`

File: app/javascript/components/shared_components/Pagination.jsx [hunk 2]
```diff
@@ -1,13 +1,27 @@
...
+  };
+
+  const disabledNext = () => {
+    if (page === totalPages) {
+      return true;
+    }
+    return false;
```
Comment (farhatahmad) [inline app/javascript/components/shared_components/Pagination.jsx:22]: `return page === totalPages`

File: app/assets/stylesheets/application.bootstrap.scss [hunk 3]
```diff
@@ -217,6 +217,12 @@ input.search-bar {
   &:focus {
     box-shadow: 0 0 0 0.25rem var(--brand-color-light);
   }
+
+  &:disabled {
+    color: gray !important;
+    pointer-events: none !important;
+    cursor: not-allowed !important;
+  }
```
Comment (farhatahmad) [inline app/assets/stylesheets/application.bootstrap.scss:225]: Theres a bootstrap `disabled` property that you can use instead https://react-bootstrap.github.io/components/buttons/#disabled-state

Comment (scouillard) [reply app/assets/stylesheets/application.bootstrap.scss:225]: It has the disabled property already but it is being overwritten by the custom button style.

**Review by farhatahmad — APPROVED** [review]

**Pull request merged**